host/tests/L3_TestCases/dsVideoDevice/dsVideoDevice_test2_SetAndGetZoomMode.py

```diff
@@ -0,0 +1,122 @@
+#!/usr/bin/env python3
+#** *****************************************************************************
+# *
+# * If not stated otherwise in this file or this component's LICENSE file the
+# * following copyright and licenses apply:
+# *
+# * Copyright 2024 RDK Management
+# *
+# * Licensed under the Apache License, Version 2.0 (the "License");
+# * you may not use this file except in compliance with the License.
+# * You may obtain a copy of the License at
+# *
+# *
+# http://www.apache.org/licenses/LICENSE-2.0
+# *
+# * Unless required by applicable law or agreed to in writing, software
+# * distributed under the License is distributed on an "AS IS" BASIS,
+# * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# * See the License for the specific language governing permissions and
+# * limitations under the License.
+# *
+#* ******************************************************************************
+
+import os
+import sys
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+sys.path.append(os.path.join(dir_path, "../../"))
+
+from L3_TestCases.dsVideoDevice.dsVideoDeviceHelperClass import dsVideoDeviceHelperClass
+
+class dsVideoDevice_test2_SetAndGetZoomMode(dsVideoDeviceHelperClass):
+    """
+    Class to perform set and get the Zoom mode on video device.
+
+    Inherits from utHelperClass to leverage common test functionalities.
+
+    Attributes:
+        testName (str): Name of the test case.
+        testSetupPath (str): Path to the test setup configuration file.
+        moduleName (str): Name of the module being tested.
+        rackDevice (str): Identifier for the device under test.
+    """
+
+    def __init__(self):
+        """
+        Initializes the test2_ZoomMode test .
+
+        Args:
+            None.
+        """
+        self.testName = "test2_ZoomMode"
+        super().__init__(self.testName, '1')
+
+       
+
+    #TODO: Current version supports only manual verification.
+    def testVerifyZoomMode(self, manual=False, videomode=False):
+        """
+        Verifies the zoom mode on specified video device.
+
+
+        Args:
+            manual (bool, optional): If True, requires manual confirmation from the user.
+                                     Defaults to False.
+
+        Returns:
+            bool: True if zoom mode is applied; otherwise, False.
+        """
+        if manual == True and videomode == True:
+            return self.testUserResponse.getUserYN(f"Is the selected Zoom mode applied in the device's output? (Y/N):")
+        elif manual == False and videomode == True:
```

Comment:
is videomode required ?

Reply:
updated

Reply:
A manual option is now provided.  The user has to confirm based on the output viewing.